### ISSUE TITLE
[Feature] make an easily deployable docker

### ISSUE THREAD

**Issue by S0ly** [Ctrlpanel-gg/panel#938]
### Feature Description

deploying pterodactyl inside a docker is possible, so why not CPGG ?

### Solution Idea

it would be nice to have a better documentation of the docker and have a image in docker hub
it would enable easy deployment and deployment on things like unraid etc... it would be a nice feature to have

**Comment by sybetjeh** [comment]
https://hub.docker.com/r/1day2die/ctrlpanel.gg - try this.

**Comment by S0ly** [comment]
it have no pull or stars, I think I seen 1d2d on the discord but is the docker working ?  any documentation on how it work ?

**Comment by S0ly** [comment]
what port, variable, etc..

**Comment by sybetjeh** [comment]
I've tried, and it seems like the image is indeed broken. I'll look into updating it and possibly adding documentation

**Comment by S0ly** [comment]
yep, thanks :D

**Comment by S0ly** [comment]
yep but its not clearly documented or in docker hub its just in some sort of construction / unfinished :)

**Comment by jameskitt616** [comment]
Is there anything holding this back? I would be very happy helping out testing this or similar to make this happen.
I look forward to a docker image hosted on any container registry (github, dockerhub or wherever), i think this would be a huge benefit for this project just having a docker container people can deploy.

Thank you very much @S0ly for making this PR! <3

**Comment by S0ly** [comment]
Hi @jameskitt616!

The standalone Docker setup should be ready, but it's made for the dev branch. It might work on the main branch, but I haven't tested it there. It's not released yet because the dev branch has issues that prevent further testing.

If you want to test it on the main branch, you'll need to build your own image using the Docker folder from the PR.

Steps to test:
1. Switch to the dev branch and get the Docker folder.
2. Navigate to `/docker/standalone`.
3. Run:
   ```sh
   docker compose up
   ```
   /!\ Please check and modify the compose values if needed.

If you want to build your own image instead of using `docker compose up`, run:
   ```sh
   docker build -t controlpanel_web:my -f ../../docker/standalone/Dockerfile ../../
   ```

**Comment by jameskitt616** [comment]
Thanks! So i checked out your branch, configured the compose and ran the ctrlpanel setup. The panel came up as expected, only weird thing is that the if i open e.g. the Overview i get an exception in the logs `Request Failed, is pterodactyl set-up correctly? - cURL error 6: Could not resolve host: api`.
I don't know why it says host: `api`, i configured it with a fully working pterodactyl instance on a subdomain.

**Comment by S0ly** [comment]
will be closed by #946
